wrappers/java/src/test/java/org/hyperledger/indy/sdk/utils/InitHelper.java

```diff
@@ -3,12 +3,10 @@
 
 import org.hyperledger.indy.sdk.LibIndy;
 
-import java.io.File;
-
 public class InitHelper {
 	public static void init() {
 
-		if (!LibIndy.isInitialized()) LibIndy.init(new File("./lib/libindy"));
+		if (!LibIndy.isInitialized()) LibIndy.init("./lib/");
```

Comment:
I agree. Actually, if libindy is in the LD path, it is already loaded automatically today, and the user doesn't have to manually call the LibIndy.init() API at all.

But for example the current unit tests (see [InitHelper.java](https://github.com/hyperledger/indy-sdk/blob/master/wrappers/java/src/test/java/org/hyperledger/indy/sdk/utils/InitHelper.java)) are loading libindy from a local directory as follows:

`LibIndy.init(new File("./lib/libindy"));
`

I agree it's preferable to load libindy from the LD path, but just in case someone doesn't want to do it, this PR provides a better, platform-independent way of doing it.